hub/client/hub_control.py

```diff
@@ -40,10 +40,11 @@ def get_dataset_path(self, tag):
         return dataset
 
     def get_credentials(self):
-
-        if self.auth_header is None:
-            token = AuthClient().get_access_token(username="public", password="")
-            self.auth_header = f"Bearer {token}"
+        self.auth_header = (
+            self.auth_header
+            if self.auth_header is not None
+            else f'Bearer {AuthClient().get_access_token(username="public", password="")}'
+        )
```

Comment:
The old way was more readable in my opinion.

Reply:
Reverted